packages/core/src/Stores/client-store.js

```diff
@@ -295,6 +295,46 @@ export default class ClientStore extends BaseStore {
         );
     }
 
+    @computed
+    get is_europe() {
+        const eu_shortcode_regex = new RegExp('^(maltainvest|malta|iom)$');
+        const eu_excluded_regex = new RegExp('^mt$');
+        const { gaming_company, financial_company } = this.landing_companies;
+        const financial_shortcode = financial_company?.shortcode;
+        const gaming_shortcode = gaming_company?.shortcode;
+        return financial_shortcode || gaming_shortcode
+            ? eu_shortcode_regex.test(financial_shortcode) || eu_shortcode_regex.test(gaming_shortcode)
+            : eu_excluded_regex.test(this.residence);
+    }
+
+    @computed
+    get should_have_real_account() {
+        return this.standpoint.iom && !this.has_any_real_account && this.residence === 'gb';
+    }
+
+    // Shows all possible landing companies of user between all
+    @computed
+    get standpoint() {
+        const result = {
+            iom: false,
+            svg: false,
+            malta: false,
+            maltainvest: false,
+        };
+        const { gaming_company, financial_company } = this.landing_companies;
+        if (gaming_company) {
+            Object.assign(result, {
+                [gaming_company.shortcode]: !!gaming_company?.shortcode,
+            });
+        }
+        if (financial_company) {
+            Object.assign(result, {
+                [financial_company.shortcode]: !!financial_company?.shortcode,
+            });
+        }
+        return result;
```

Comment:
why do we need this? can't we just check client.landing_company_shortcode === 'iom' etc?

Reply:
@negarn as I said earlier, based on [this flow](https://miro.com/app/board/o9J_kvOgFCg=/), there are different cases between MLT/MF, Only MLT, and Only MF users. I created this object to track all of them. checking a single string is not covering all cases unless you have a better idea.

Reply:
@mahdiaryayi-fs that's fine, let's go with this